pom.xml

```diff
@@ -793,13 +793,29 @@ under the License.</licenseText>
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-pmd-plugin</artifactId>
```

Comment:
its already there and in parent just the same. Sure we align on proper tooling. Ruleset will evolve with existing issues being fixed:
- https://github.com/apache/maven/pull/2402
-